### DISCUSSION TITLE
Set PhotoShop path as clipping path

### DISCUSSION THREAD

**Discussion by MiHuIT** [ImageMagick/ImageMagick#5767] [Help]
hello all,
is it possible in ImageMagick to set a "normal" PhotoShop path as a ClippingPath?
If this is not possible in ImageMagick, do you know another way to do this?

Thanks for your help

**Comment by fmw42** [comment]
I  believe that Imagemagick treats all paths the same. Normally Imagemagick just takes the first path by default when using -clip.

But if you want to select a path that is not the first in the list for clipping then  if you want to choose a path (for example #6), you can do:

```
magick identify -quiet -format "%[8BIM:1999,2998:#6]" clippingpath.tif |\
magick -quiet clippingpath.tif \( - -negate \) \
-alpha off -compose copy_opacity -composite \
-define tiff:alpha=associated \
clippingpath.png
```

**Comment by MiHuIT** [comment]
Thank you for the quick reply.
Unfortunately, the use of the path is not the problem.

I pass the image via an API to a third party system of the client.
This system requires an activated clipping path, but I'm not quite sure why (I assume due to automatic product placement). Now I must somehow find a way to declare this path (the name is always unique) as an active clipping path.

Do you see a way to do this?

Thank you and greetings

**Reply by fmw42** [reply]
Imagemagick cannot define Photoshop clipping paths as far as I know.  It does not distinguish between type of paths.

I am not sure I understand what you want. So I will leave it to the IM developers for further comments.
